pkg/console/tlsconfig/tlsconfig_test.go

```diff
@@ -214,25 +221,59 @@ var _ = Describe("TlsConfig", func() {
 				Expect(err).ToNot(HaveOccurred())
 			}()
 
-			Eventually(func() error {
-				_, err := configWatch.GetConfig()
-				return err
-			}, 1*time.Second, 100*time.Millisecond).Should(MatchError(ContainSubstring("failed to load certificate")))
+			mockWatch.Trigger(certAndKeyDir)
+
+			_, err := configWatch.GetConfig()
+			Expect(err).To(MatchError(ContainSubstring("failed to load certificate")))
```

Comment:
Do we have a better way to test this error than string matching?

Reply:
No, because the error is created using `fmt.Errorf()`.